samples/tab-add-in-combined/nodejs/add-in/webpack.config.js

```diff
@@ -0,0 +1,105 @@
+/* eslint-disable no-undef */
+
+const devCerts = require("office-addin-dev-certs");
+const CopyWebpackPlugin = require("copy-webpack-plugin");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+
+const urlDev = "https://localhost:3000/";
```

Comment:
Should we mention this in our Readme file?

Reply:
Will cover in different new PR